package.json

```diff
@@ -54,6 +55,7 @@
     "@testing-library/react": "^11.2.5",
     "@types/node": "^14.14.33",
     "@types/pem": "^1.9.5",
+    "@types/pumpify": "^1.4.1",
```

Comment:
Note: These types are needed for the type definition for `pino-applicationinsights`